Fix typos

rex4539 opened the pull request.

rex4539 committed:
Fix typos

rex4539 commented:
@sisou Could you please take a look?

jsdanielh commented on scripts/devnet/node.py:
```diff
@@ -319,7 +319,7 @@ def poll(self):
         :return: The return code of the process if it ended, None if it hasn't
             finished
-        :rtype: int ot None
+        :rtype: int to None
```
```suggestion
        :rtype: int or None
```

rex4539 replied:
Fixed and rebased.

jsdanielh reviewed (changes requested):
Thanks for the PR. I only noted a wrong fix. Could you fix it?

jsdanielh committed:
lib: Fix compilation error uncovered after fixing a typo

Fix compilation error that was uncovered after fixing a typo inside
the `lib` subcrate when the `launcher` and `panic` features are
enabled.

jsdanielh commented:
@rex4539 Will push an additional commit since you just uncovered a compilation error that was hidden under the `fature` typo.
